### PR TITLE
*: Support avg for localstore xapi

### PR DESCRIPTION
**Pull request by shenli**
@coocood @zimulala PTAL
@BusyJay

### CODE REVIEW

File: executor/aggregate_xapi_test.go [hunk 1]
```diff
@@ -274,3 +274,102 @@ func (s *testAggFuncSuite) TestXAPISum(c *C) {
...
+	c.Assert(err, IsNil)
+	c.Assert(val, testutil.DatumEquals, types.NewDecimalDatum(mysql.NewDecimalFromInt(int64(5), 0)))
+	// Second row: 21
+	row, err = agg.Next()
+	c.Assert(err, IsNil)
+	c.Assert(row, NotNil)
+	val, err = evaluator.Eval(ctx, fc)
+	c.Assert(err, IsNil)
+	c.Assert(val, testutil.DatumEquals, types.NewDecimalDatum(mysql.NewDecimalFromInt(int64(21), 0)))
+	// Third row: 31
```
Comment (zimulala) [inline executor/aggregate_xapi_test.go]: Add a decimal type of test.

**Comment by shenli** [comment]
PTAL @zimulala

File: executor/aggregate_xapi_test.go [hunk 2]
```diff
@@ -274,3 +274,105 @@ func (s *testAggFuncSuite) TestXAPISum(c *C) {
...
+	val, err = evaluator.Eval(ctx, fc)
+	c.Assert(err, IsNil)
+	c.Assert(val, testutil.DatumEquals, types.NewDecimalDatum(mysql.NewDecimalFromInt(int64(21), 0)))
+	// Third row: 16.5000
+	row, err = agg.Next()
+	c.Assert(err, IsNil)
+	c.Assert(row, NotNil)
+	val, err = evaluator.Eval(ctx, fc)
+	c.Assert(err, IsNil)
+	c.Assert(val, testutil.DatumEquals, types.NewDecimalDatum(mysql.NewDecimalFromFloat(float64(16.5))))
```
Comment (zimulala) [inline executor/aggregate_xapi_test.go]: Here the val is 16.5000 ?

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
@zimulala PTAL

**Comment by zimulala** [comment]
LGTM

**Pull request merged**